source/sass/buyers-guide/views/category.scss

```diff
@@ -39,6 +39,8 @@
         padding-right: 3%;
       }
 
+      @import "../fragments/_draft_product.scss";
```

Comment:
We shouldn't `@import` within a declaration block as it copies and includes the same set of rules every time we do this.

<img width="954" alt="image" src="https://user-images.githubusercontent.com/2896608/51068096-233a9980-15ce-11e9-86c4-62c9e1c353a8.png">

Reply:
With the accessibility concern above I think we can probably just have CSS rules for `.draft-note` and have JS inject a `draft-note` node wherever appropriate.

Reply:
Whether you use an import or a mixin, the final CSS has the code duplicated as many times as necessary  (both includes and mixins are sass constructs, not CSS constructs) so unless I'm missing something that got landed in CSS itself that allows for reusables now, using one or the other has no effect on the final CSS, only on SASS (in which case this is a normal partial, detected by SASS through the leading underscore in the file name)

Reply:
I'm with Mavis on this one- an `@import` block in the middle of the sheet feels weird, for some reason. I might even argue that this should just be a class that's applied to a bunch of things in HTML rather than being imported in multiple CSS files. Though I haven't looked extremely closely at where/how this is used.